src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -158,6 +161,16 @@
                 "ARRAY_INIT",
                 // these are covered by GenericWhitespaceCheck
                 "WILDCARD_TYPE", "GENERIC_END", "GENERIC_START").collect(Collectors.toSet()));
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("EmptyBlock", Stream.of(
+                // state of the configuration when test was made until
+                "INSTANCE_INIT", "LITERAL_DO", "LITERAL_ELSE", "LITERAL_FINALLY", "LITERAL_FOR",
+                "LITERAL_IF", "LITERAL_SWITCH", "LITERAL_SYNCHRONIZED", "LITERAL_TRY",
+                "LITERAL_WHILE", "STATIC_INIT")
+                .collect(Collectors.toSet()));
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("RightCurly", Stream.of(
+                // state of the configuration when test was made until
+                "LITERAL_CATCH", "LITERAL_ELSE", "LITERAL_FINALLY", "LITERAL_IF", "LITERAL_TRY")
```

Comment:
default of RightCurly is `same`.
I think we should be same https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L185

Am I miss smth ?